contrib/test/integration/build/cri-o.yml

```diff
@@ -1,42 +1,42 @@
 ---
 
-- name: stat the expected cri-o directory
+- name: stat the expected cri-o directory and Makefile exists
   stat:
-    path: "{{ ansible_env.GOPATH }}/src/github.com/kubernetes-incubator/cri-o"
+    path: "{{ cri_o_dest_path }}/Makefile"
   register: dir_stat
```

Comment:
Given that the test sense has changed (stat a file now, not a dir), could you rename `dir_stat` please?

Reply:
Excellent suggestion...consider it done...